pkg/mbw/monitor/controller.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2022 The Katalyst Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package monitor
+
+import (
+	"fmt"
+
+	"github.com/kubewharf/katalyst-core/pkg/mbw/utils/rdt"
+	"github.com/kubewharf/katalyst-core/pkg/util/general"
+)
+
+type MB_CONTROL_ACTION int
+
+const (
+	BW_LEN                = 11
+	MBA_COS_MAX           = 16 // AMD supports up to 16 cos per CCD, while Intel supports up to 16 cos per socket
+	L3QOS_BW_CONTROL_BASE = 0xc0000200
+
+	MEMORY_BANDWIDTH_THRESHOLD_PHYSICAL_NUMA = 140000 // fixed threshold 140GBps each physical numa node
```

Comment:
I used this const for testing. Its value should be assigned from configuration and machine dependent. If this PR focuses on mbw monitoring only, this file can be removed.